java/src/Ice/src/main/java/com/zeroc/Ice/ObjectAdapterI.java

```diff
@@ -375,48 +390,59 @@ public synchronized Map<String, Object> removeAllFacets(Identity ident) {
   }
 
   @Override
-  public synchronized com.zeroc.Ice.Object removeDefaultServant(String category) {
+  public synchronized Object removeDefaultServant(String category) {
     checkForDeactivation();
 
     return _servantManager.removeDefaultServant(category);
   }
 
   @Override
-  public com.zeroc.Ice.Object find(Identity ident) {
+  public Object find(Identity ident) {
     return findFacet(ident, "");
   }
 
   @Override
-  public synchronized com.zeroc.Ice.Object findFacet(Identity ident, String facet) {
+  public synchronized Object findFacet(Identity ident, String facet) {
     checkForDeactivation();
     checkIdentity(ident);
 
     return _servantManager.findServant(ident, facet);
   }
 
   @Override
-  public synchronized java.util.Map<String, com.zeroc.Ice.Object> findAllFacets(Identity ident) {
+  public synchronized java.util.Map<String, Object> findAllFacets(Identity ident) {
     checkForDeactivation();
     checkIdentity(ident);
 
     return _servantManager.findAllFacets(ident);
   }
 
   @Override
-  public synchronized com.zeroc.Ice.Object findByProxy(ObjectPrx proxy) {
+  public synchronized Object findByProxy(ObjectPrx proxy) {
     checkForDeactivation();
 
     com.zeroc.IceInternal.Reference ref = ((_ObjectPrxI) proxy)._getReference();
     return findFacet(ref.getIdentity(), ref.getFacet());
   }
 
   @Override
-  public synchronized com.zeroc.Ice.Object findDefaultServant(String category) {
+  public synchronized Object findDefaultServant(String category) {
     checkForDeactivation();
 
     return _servantManager.findDefaultServant(category);
   }
 
+  @Override
+  public synchronized Object dispatchPipeline() {
```

Comment:
Not that it's wrong, I'm just curious, why upgrade the visibility here?
In the interface this method is only "package-private" (by default), and we upgrade this method to "public" visibility.

I'm curious, but also I feel like it's best left package-private to prevent users from accidentally calling this (thinking it's a simple getter) and then locking up their `middlewareStack`.

Reply:
`dispatchPipeline` is a method of public interface ObjectAdapter.

`public` is required on the implementation:
```
Cannot reduce the visibility of the inherited method from ObjectAdapterJava(67109273)
```